Xamarin.Forms.Platform.MacOS/ImageSourceHandlers.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using AppKit;
+using Foundation;
+
+namespace Xamarin.Forms.Platform.MacOS
```

Comment:
not that easy they aren't in the same file , (on iOS are inside the ImageRenderer) i can share this file with iOS, and delete the iOS code ..